include/deal.II/multigrid/multigrid.h

```diff
@@ -478,7 +478,7 @@ class Multigrid : public Subscriptor
    */
   SmartPointer<const MGMatrixBase<VectorType>, Multigrid<VectorType>> edge_up;
 
-  template <int dim, class OtherVectorType, class TRANSFER>
+  template <int dim, typename OtherVectorType, class TRANSFER>
```

Comment:
Do you want to address this one, too?
```suggestion
  template <int dim, typename OtherVectorType, typename TransferType>
```

Reply:
Might as well. I also fixed `class DATA` in `mg_smoother.h`.